### PR TITLE
Mkdir: explicit umask and sequential tests

### PR DESCRIPTION
**Pull request by shinhs0506**
1. explicitly set umask when testing file mode (https://github.com/uutils/coreutils/pull/4876#issuecomment-1560070035)
2. run tests sequentially since tests that do set umask cause other tests to fail

### CODE REVIEW

**Comment by cakebaker** [comment]
Why don't you set umask in `test_symbolic_mode` and `test_multi_symbolic`?

**Comment by cakebaker** [comment]
Never mind, you set the permissions in an "absolute" way.

File: tests/by-util/test_mkdir.rs [hunk 1]
```diff
@@ -1,8 +1,12 @@
 use crate::common::util::TestScenario;
 #[cfg(not(windows))]
 use libc::{mode_t, umask};
+use once_cell::sync::Lazy;
 #[cfg(not(windows))]
 use std::os::unix::fs::PermissionsExt;
+use std::sync::Mutex;
+
+static TEST_MUTEX: Lazy<Mutex<()>> = Lazy::new(|| Mutex::new(()));
```
Comment (sylvestre) [inline tests/by-util/test_mkdir.rs:9]: please document why it has to be done :)
thanks

**Comment by cakebaker** [comment]
Thanks!

**Pull request merged**